internal/cmd/cmd.go

```diff
@@ -163,14 +163,13 @@ $ infra use development.kube-system`,
 			parts := strings.Split(destination, ".")
 
 			if parts[0] == "kubernetes" {
-				if len(parts) > 2 {
-					return kubernetesSetContext(parts[1], parts[2])
+				if len(parts) > 1 {
+					return kubernetesSetContext(parts[0], parts[1])
 				}
 
-				return kubernetesSetContext(parts[1], "")
+				return kubernetesSetContext(parts[0], "")
 			}
```

Comment:
This entire block is no longer necessary

Reply:
Good catch, thanks!